server/src/services/standupScheduler.ts

```diff
@@ -0,0 +1,49 @@
+import { slackApp } from "..";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import { logger } from "../logger";
+import addStandupQueue from "../queues/standupQueue";
+const initiateStandupOnSchedule = require("./postMessage");
+import { standupsAvailable } from "./standups.service";
+import { timeConversion } from "../utils/timeConversion";
+
+export const scheduleStandup = async (
+  channelId: string,
+  channelName: string,
+  members: {}[]
+): Promise<void> => {
+  const token = slackApp.client.token;
+  const client = slackApp.client;
+
+  members?.forEach(async (user: any) => {
+    const channel = user.slackMemberId;
+    initiateStandupOnSchedule({
+      token,
+      channel,
+      client,
+      channelId,
+      channelName,
+    });
+  });
+};
+
+export const scheduledStandups = async () => {
+  logger.debug(
+    "scheduled standup function ran that schedules the standup present in current hour"
+  );
+
+  let standupsAvailableThisHour: any = await standupsAvailable();
```

Comment:
check the type. Do not use `any`
```suggestion
  const standupsAvailableThisHour: any = await standupsAvailable();
```